packages/api-reference/src/components/Content/Tag/Endpoints.vue

```diff
@@ -17,19 +17,21 @@ import {
 
 const props = defineProps<{ id?: string; tag: Tag }>()
 
+const emit = defineEmits<{
+  (event: 'observeAndNavigate', value: string): void
+}>()
```

Comment:
```suggestion
const emit = defineEmits<{
  [observeAndNavigate: [value: string]
}>()
```
While there's absolutely nothing wrong with that syntax, this is the [cool new way of doing emits](https://vuejs.org/guide/typescript/composition-api#typing-component-emits)


![image](https://github.com/scalar/scalar/assets/2039539/3b1d0ca1-4c35-48c7-a573-5eb497fdac76)

Reply:
wow til! thank you